### PR TITLE
Fix code for codacy style

### PR DESCRIPTION
**Pull request by simoexpo**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -74,11 +71,11 @@ Community leaders will follow these Community Impact Guidelines in determining t
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 2.0,
-available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+available at <https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
 
 Community Impact Guidelines were inspired by [Mozilla's code of conduct enforcement ladder](https://github.com/mozilla/diversity).
 
-[homepage]: https://www.contributor-covenant.org
```
Comment (massimosiani) [inline CODE_OF_CONDUCT.md]: why moved below?

Comment (simoexpo) [reply CODE_OF_CONDUCT.md]: It was a codacy warning. Something like `definition should be at the end of the file`

Comment (massimosiani) [reply CODE_OF_CONDUCT.md]: did you check the resulting output? I think we should escape the brackets instead of blindly obey to suggestions :)

Comment (massimosiani) [reply CODE_OF_CONDUCT.md]: my fault.

**Pull request merged**